src/player/index.ts

```diff
@@ -144,6 +150,70 @@ export async function focusPlayer() {
   showDocument(currentThread.uri, currentThread.range);
 }
 
+var onigPath = path.join(__dirname, "./wasm/onig.wasm");
```

Comment:
loading the wasm, parsing the language, and creating the registry can be added in extension.ts. so that we don't have to load them again every time we start the step.

Reply:
Yeah, I figured there's a better way! Thanks for the feedback.